### PR TITLE
Listen to the installation part

### PR DESCRIPTION
**Pull request by Bocz3k**
This PR makes the user listen to the installation part in... French

what am i doing

### CODE REVIEW

**Comment by TodePond** [comment]
I like it but I wonder if there's some way of embedding it within the page? I know you can embed videos, for example

File: README.md [hunk 1]
```diff
@@ -171,11 +171,7 @@ Loops are a complicated relic of archaic programming languages. In WhenWillProgr
 
 ## Installation
 
-To install WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite to your command line, first install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer.<br>
-To install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer, install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer installer.
-
-**New for 2022!**<br>
-Due to the complicated installation process, you can now install the 'Create WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite App' app that installs everything for you!
+To listen to this part [download the .mp3 file](files/installation.mp3)
```
Comment (Joshix-1) [inline README.md:174]:
```suggestion
To listen to this part ![download the .mp3 file](files/installation.mp3)
```

Comment (TodePond) [reply README.md]: Ack seems like that still didn't do it

Comment (TodePond) [reply README.md]: Ah
image

Maybe it's worth converting it into a video file. then people can just click play

Comment (Joshix-1) [reply README.md]: installation.webm

**Comment by Bocz3k** [comment]
I've broke it, and if i want to unbreak it, I will have to probably be stuck with this repo. So for now it will be broken

**Pull request merged**